eng/Version.Details.xml

```diff
@@ -27,36 +27,36 @@
       <Sha>1127689f262d52ea8ff68ef03d706fa62b3b40a1</Sha>
       <RepoName>linker</RepoName>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.Runtime.CoreCLR" Version="3.1.16-servicing.21262.5" CoherentParentDependency="Microsoft.Private.CoreFx.NETCoreApp">
-      <Uri>https://github.com/dotnet/coreclr</Uri>
-      <Sha>b820e148c5ded6a37cb3ddb38b88b54210189562</Sha>
+    <Dependency Name="Microsoft.NETCore.Runtime.CoreCLR" Version="3.1.17-servicing.21315.6" CoherentParentDependency="Microsoft.Private.CoreFx.NETCoreApp">
+      <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-coreclr</Uri>
```

Comment:
These shouldn't be the internal repos should they?

Reply:
Correct, thanks.